### DISCUSSION TITLE
Add a zoom/full screen button for image of tokens

### DISCUSSION THREAD

**Discussion by Agilulfo1820** [blockscout/blockscout#3636] [Ideas]
On the ERC 721 token page, we can see the image of the token taken from the metadata. I think it would be useful to have a button to see the image in full-screen mode. What do you think?

**Comment by vbaranov** [comment]
That is a good idea! Or even to make image clickable and open its original link in a new tab. If you can prepare a PR, I'd love to review/merge it to master branch.

**Reply by Agilulfo1820** [reply]
I'll try this weekend, but I never used elixir nor erlang before so I hope I'll be able to grasp it fast.
